components/generic/ImageWithAttribution.vue

```diff
@@ -1,60 +1,106 @@
 <template>
-  <div>
-    <figure>
-      <OptimisedImage
```

Comment:
ImageWithAttribution needs to keep using OptimisedImage, so that images are lazy-loaded, and optimised via the Contentful Image API.